tools/accuracy_checker/configs/mixnet-l.yml

```diff
@@ -0,0 +1,85 @@
+models:
+  - name: mixnet-l
+
+    launchers:
+      - framework: tf
+        model:   public/mixnet-l/mixnet-l/model.ckpt.meta
+        adapter: classification
+        output_names:
+          - logits
+        inputs:
+          - name: IteratorGetNext
+            type: INPUT
+
+    datasets:
+        
+      - name: imagenet_1000_classes
+        annotation_conversion:
+           converter: imagenet
+           annotation_file: val.txt
+        
+        data_source: ILSVRC2012_img_val
```

Comment:
please remove these sections, we apply dataset definitions for getting this info